src/api/proto/vispb/vis.proto

```diff
@@ -308,6 +308,17 @@ message TimeseriesChart {
   Axis y_axis = 4;
 }
 
+// StatChart The visual spec for a basic stat chart. Shows the latest value in the table as a big
+// number with units.
+message StatChart {
+  message Stat {
+    // The (numeric) data field that will be used as the Y axis
+    string value = 1;
+  }
+  Stat stat = 1;
```

Comment:
Can you add comments for each of these fields? We try to do so in our protos even if it seems like it should be obvious.

Reply:
Done.